README.md

```diff
@@ -93,10 +104,15 @@ By default, this runs a reproducible build in a [Docker](https://docs.docker.com
 - requires that `Cargo.lock` of project is created (e.g. via `cargo update`) and added to git 
 - will use configuration in `[package.metadata.near.reproducible_build]` section of contract's `Cargo.toml`
 
-`--no-docker` flag can be used to perform a regular build with rust toolchain installed onto host, running the `cargo-near` cli. *NO*-Docker builds run against actual state of code in filesystem and not against a version, committed to source control.   
+`--no-docker` flag can be used to perform a regular build with rust toolchain installed onto host, running the `cargo-near` cli. 
+  - *NO*-Docker builds run against actual state of code in filesystem and not against a version, committed to source control.   
 
 `--no-locked` flag is allowed in *NO*-Docker builds, e.g. to generate a `Cargo.lock` *and* simultaneously build the contract.
 
+`--no-locked` flag is allowed in Docker builds, but 
+  - such builds are considered effectively DEMO-mode
```

Comment:
Before this feature, we had 4 years of NEAR living just fine with non-reproducible builds, let's not discourage it too much. Such contracts are still valid contracts and can hold millions $ in assets.

```suggestion
```